### ISSUE TITLE
Missing base path in JSON-LD response

### ISSUE THREAD

**Issue by csarven** [solid/node-solid-server#476]
`curl -iH"Accept: application/ld+json" https://example.org/container/` returns a JSON-LD body like:

```json
[
  {
    "@id": "https://example.org",
    "http://www.w3.org/ns/ldp#contains": [
      {
        "@id": "https://example.orgdc354280-cb7c-11e6-9c08-3d5cf446a570"
      }
    ]
  }
]
```

The expected response is:
```json
[
  {
    "@id": "https://example.org/container/",
    "http://www.w3.org/ns/ldp#contains": [
      {
        "@id": "https://example.org/container/dc354280-cb7c-11e6-9c08-3d5cf446a570"
      }
    ]
  }
]
```

**Comment by dmitrizagidulin** [comment]
Wow, good catch

**Comment by dmitrizagidulin** [comment]
This bug actually also affects all the other serializations - Turtle, NTriples, etc.

**Comment by dmitrizagidulin** [comment]
Since the fix involves fairly extensive LDP changes, I will fix this as part of PR #484.

**Comment by dmitrizagidulin** [comment]
Addressed by PR https://github.com/solid/node-solid-server/pull/496
